### PR TITLE
Swithch to gicv3

### PR DESCRIPTION
**Pull request by dsemenets**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -9,12 +9,13 @@ include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
...
-    DEPENDS ${ZEPHYR_FINAL_EXECUTABLE}
-    COMMAND ${UTILS_SCRIPTS_DIR}/fix_dtb.sh ${UTILS_SCRIPTS_DIR}/gicv2.tmpl ${ZEPHYR_BINARY_DIR}/virt_gicv2.dtb ${ZEPHYR_BINARY_DIR}/zephyr.bin)
+    add_custom_target(
+        fix_dtb
+        DEPENDS ${ZEPHYR_FINAL_EXECUTABLE}
+        COMMAND ${UTILS_SCRIPTS_DIR}/fix_dtb.sh ${UTILS_SCRIPTS_DIR}/gicv${QEMU_GIC_VERSION}.tmpl ${ZEPHYR_BINARY_DIR}/virt_gicv${QEMU_GIC_VERSION}.dtb ${ZEPHYR_BINARY_DIR}/zephyr.bin
+    )
 else()
-  message(WARNING "No dtc program")
+    message(WARNING "No dtc progren")
```
Comment (oleksiimoisieiev) [inline CMakeLists.txt]: progren -> program. 
Also - do we really need this intendation changes?

**Comment by oleksiimoisieiev** [comment]
Why do we need dtb files here?
Also please update commit messages.
And make gic2 gic3 configurable

File: CMakeLists.txt (hunk 1 above)
Comment (dsemenets) [reply CMakeLists.txt]: This is misprint

Comment (oleksiimoisieiev) [reply CMakeLists.txt]: please remove it from commit

**Pull request closed without merging**